sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -267,19 +275,28 @@ function toCorePolicy(
 
 function toPublicPolicy(policy: CoreCertificatePolicy = {}): CertificatePolicy {
   let certificatePolicy: CertificatePolicy = {
-    lifetimeActions: policy.lifetimeActions,
-    contentType: policy.secretProperties ? policy.secretProperties.contentType : undefined
+    lifetimeActions: policy.lifetimeActions
+      ? policy.lifetimeActions.map((action) => ({
+          action: action.action ? action.action.actionType : undefined,
```

Comment:
Should this also change (re: "actionType")?

Reply:
Good catch

Reply:
We chatted a bit off-line. This is actually part of the repack for the core service, so this is correct.